talk/basicconcepts/references.tex

```diff
@@ -29,24 +29,23 @@
   \frametitlecpp[98]{Pointers vs References}
   \begin{block}{Specificities of reference}
     \begin{itemize}
-    \item natural syntax
-    \item must be assigned when defined, cannot be \mintinline{cpp}{nullptr}
-    \item cannot be reassigned
-    \item non-const references to temporary objects are not allowed
+    \item Natural syntax
+    \item Cannot be \mintinline{cpp}{nullptr}
+    \item Must be assigned when defined, cannot be reassigned
```

Comment:
When talking about references, there is also to term "to reseat" a reference/pointer instead of reassign or reset. Should we mention it somewhere is it too much C++ lingo?

Reply:
I think it's out of scope for this pull request, but in any case I think that for the basic part, it's better to leave that out.